lib/process-services-cloud/src/lib/form/components/widgets/dropdown/dropdown-cloud.widget.html

```diff
@@ -22,15 +20,16 @@
                 [title]="field.tooltip"
                 panelClass="adf-select-filter"
                 [multiple]="field.hasMultipleValues"
-                [required]="isRequired()"
+                [disabled]="field.readOnly"
+                [required]="field.required "
             >
                 <adf-select-filter-input *ngIf="showInputFilter" (change)="filter$.next($event)" />
 
                 <mat-option *ngFor="let opt of list$ | async" [value]="opt" [id]="opt.id">{{opt.name}}</mat-option>
                 <mat-option id="readonlyOption" *ngIf="isReadOnlyType" [value]="field.value">{{field.value}}</mat-option>
             </mat-select>
         </mat-form-field>
-        <div *ngIf="!previewState && !field.readOnly">
```

Comment:
is this removal intended? it's replaced with the CSS class name, which may not have the same behaviour